src/semgrep_agent/findings.py

```diff
@@ -38,13 +38,13 @@ class Finding:
     commit_date = attr.ib(
         type=Optional[datetime], default=None, hash=None, eq=False, kw_only=True
     )
-    metadata = attr.ib(
-        type=Optional[Mapping[str, Any]],
-        default=None,
-        hash=None,
-        eq=False,
-        kw_only=True,
-    )
+    metadata = attr.ib(type=Mapping[str, Any], hash=None, eq=False, kw_only=True,)
```

Comment:
```suggestion
    metadata = attr.ib(type=Mapping[str, Any], hash=None, eq=False, kw_only=True)
```

Teeny tiny change but might as well 😇